src/logic/operate.js

```diff
@@ -0,0 +1,31 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const nOne = Big(numberOne);
+  const nTwo = Big(numberTwo);
+  let result = null;
+
+  switch (operation) {
+    case '-':
+      result = nOne.minus(nTwo);
+      break;
+    case '+':
+      result = nOne.plus(nTwo);
+      break;
+    case '*':
+      result = nOne.mul(nTwo);
+      break;
+    case '/':
+      result = nTwo === 0 ? 'undefined' : nOne.div(nTwo);
+      break;
+    case '%':
+      result = nOne.div(100);
```

Comment:
- [ ] Instead of using `div` here, try using the [`mod`](http://mikemcl.github.io/big.js/#mod) function from `Big.js` module to work with percentages.